packages/dashmate/src/util/resolvePublicIpV4.js

```diff
@@ -0,0 +1,12 @@
+const publicIp = require('public-ip');
+const wait = require('./wait');
+
+async function resolvePublicIpV4() {
```

Comment:
We should use it here as well https://github.com/dashpay/platform/blob/48ee2353c4c993b936e8bd8febb2d4ff84ffe4af/packages/dashmate/src/listr/prompts/createIpAndPortsForm.js#L64